src/chain/async_tree.rs

```diff
@@ -114,6 +115,17 @@ pub struct Config<TAsync> {
 
     /// After an asynchronous operation fails, retry after this given duration.
     pub retry_after_failed: Duration,
+
+    /// Number of elements to initially allocate to store blocks.
```

Comment:
In the Rust ecosystem, the word `capacity` is always a number of elements.
The goal is not so much to have a specific amount of memory being used, but rather to allocate enough space at the beginning in order to avoid the cost of relocating containers when they're too small.
For this reason, the size of the element doesn't matter, you just pass the number of elements that you expect to store.